packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown.tsx

```diff
@@ -86,11 +94,26 @@ export class SelectWrapperDropdown {
       // therefore we do it here via attribute
       ['hidden', 'disabled', 'selected']
     );
+    this.detectNativePopoverCase();
   }
 
   public componentDidRender(): void {
     if (this.isOpen) {
       handleScroll(this.listElement, getHighlightedOptionMapIndex(this.optionMaps));
+
+      if (this.isNativePopover) {
+        this.popoverElement.showPopover();
+        updateNativePopoverSelectStyles(
+          this.host,
+          this.optionMaps,
+          this.popoverElement,
+          this.buttonElement,
+          this.direction
+        );
+        addNativePopoverScrollAndResizeListeners(this.host, this.parentTableElement, this.popoverElement);
+      }
+    } else if (this.isNativePopover) {
+      this.popoverElement.hidePopover();
```

Comment:
Same could probably be achieved like this:
```suggestion
    } else {
      this.popoverElement?.hidePopover();
```

Reply:
Fixed